content/developer/reference/backend/testing.rst

```diff
@@ -819,35 +840,39 @@ driven explicitly by the user) it's more complicated when running
 "test" tours, or when running tours through the test suite. In that
 case there are two main tricks:
 
-- Have a step with a ``run() { debugger; }`` action.
+- A step property ``break: true,`` in debug mode (debug=True).
 
-  This can be added to an existing step, or can be a new dedicated
-  step. Once the step's **trigger** is matched, the execution will
-  stop all javascript execution.
+  This adds a debugger breakpoint at the start of the step.
+  You can then add your own wherever you need.
 
   **Advantages**
     - very simple
-    - the tour restarts as soon as you resume execution
+    - the tour continues as soon as you resume execution
   **Drawbacks**
     - page interaction is limited as all javascript is blocked
-    - debugging the inside of the tour manager is not very useful
-- Add a step with a trigger which never succeeds and a very long
-  ``timeout``.
+    - a step property ``pause: true,`` in debug mode (debug=True).
```

Comment:
Seems a bit confusing, `pause:true` is a drawback of `break:true` ? :thinking: 
From what i read in your commit message, isn't it another method for debugging steps ?

Reply:
Yes, there are two attributes to make debugging a step easier. The first is break:true. This starts the step with a breakpoint (debugger in javascript console), which then allows you to see what is happening in the javascript code. The second is pause:true. This is triggered at the end of the stage and "simply" freezes the tour (not a debugger in javascript console). This makes it easier to interact with the DOM and see if the step did what we wanted it to do.